consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/blockcreation/MergeMiningCoordinator.java

```diff
@@ -71,6 +71,14 @@ default boolean isCompatibleWithEngineApi() {
    */
   BlockProcessingResult validateBlock(final Block block);
 
+  /**
+   * Validate a proposed block.
```

Comment:
nit: Worth mentioning the fact that it won't add to bad block report here?

Reply:
Adding details that this doesn't report a bad block would be helpful, it's not clear from the name why we need a different method otherwise

Reply:
done